pkg/transforms/policy.go

```diff
@@ -37,6 +41,99 @@ func PolicyResourceBuilder(p *p.Policy) *PolicyResource {
 	if okns && okpp {
 		node.Properties["_parentPolicy"] = pnamespace + "/" + ppolicy
 	}
+
+	return &PolicyResource{node: node}
+}
+
+// For cert, config, operator policies.
+// This function returns `annotations`, `_isExternal` for `source`,
+// and `severity`, `compliant`, and `remediationAction`.
+func getPolicyCommonProperties(c *unstructured.Unstructured, node Node) Node {
+	annos := c.GetAnnotations()
+	// This annotation is large and useless
+	delete(annos, "kubectl.kubernetes.io/last-applied-configuration")
+
+	node.Properties["annotations"] = annos
+
+	node.Properties["_isExternal"] = false
```

Comment:
Would it work if you add this property only when `true`?
It could help to reduce some data. It's only a small impact, but everything counts.

Reply:
Well, ignore this comment. I don't think it is worth the additional complexity.
I wouldn't do the same for searchable properties (without the underscore) because users expect to search for `property=false`.